pkg/dohdec/types/dot.d.ts

```diff
@@ -160,10 +160,13 @@ export type DOT_LookupOptions = {
      */
     decode?: boolean;
     /**
-     * Request DNSSec records.  Currently
-     * requires `json: false`.
```

Comment:
This change is unrelated strictly speaking, but I noticed it works with JSON too now so I thought I might as well update it whilst I'm there.

Reply:
Never mind, it does require `json=false`.